infra/cmake/packages/Opencl_HeadersSourceConfig.cmake

```diff
@@ -0,0 +1,22 @@
+function(_Opencl_HeadersSource_import)
+  if(NOT DOWNLOAD_OPENCL_HEADERS)
+    set(Opencl_HeadersSource_FOUND FALSE PARENT_SCOPE)
+    return()
+  endif(NOT DOWNLOAD_OPENCL_HEADERS)
+
+  nnas_include(ExternalSourceTools)
+  nnas_include(OptionTools)
+
+  envoption(EXTERNAL_DOWNLOAD_SERVER "https://github.com")
+  envoption(OPENCL_HEADERS_URL ${EXTERNAL_DOWNLOAD_SERVER}/KhronosGroup/OpenCL-Headers/archive/v2021.04.29.tar.gz)
```

Comment:
This does not work on tizen gbs build since gbs build does not allow to download on build. Thus, we need to put tar files into ONE/packaging and untar during gbs build. Please see nnfw.spec file for detail.

Reply:
oh thanks.. i will more check.

Reply:
or ask me via offline :)

Reply:
Thanks, I think it was added in a new commit. please check. If this is wrong... I'll go ask them offline.

Reply:
Please add stamp file into zipped file.